src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  --white: #fff;
+  --black: #000;
+  --blue: #00acdc;
```

Comment:
The variable name should tell us about their purpose, for example `--main-color: #00acdc;` as if the designer changes the palette to use red instead it will be confusing